src/jack.c

```diff
@@ -318,44 +318,56 @@ jalv_backend_init(Jalv* jalv)
 {
 	jack_client_t* client = NULL;
 
-	/* Determine the name of the JACK client */
-	char* jack_name = NULL;
-	if (jalv->opts.name) {
-		/* Name given on command line */
-		jack_name = jalv_strdup(jalv->opts.name);
+	/* If the backend is already allocated
+	 * also the client should be available
+	 * becasue it is used as an internal JACK client
+	 */
+	if (jalv->backend != NULL) {
```

Comment:
nit: Jalv uses `if (jalv->backend)` and `if (!some_pointer)` and so on, but no big deal.